### ISSUE TITLE
[Task]: Revert Crowdin workflow once stardust-develop is merged into develop

### ISSUE THREAD

**Issue by cvarley100** [iotaledger/firefly#4002]
### Task description

We have updated the crowdin workflow for the stardust-develop branch. 

Once merged into develop, we will need to revert the changes in Crowdin.

### Requirements

Update crowdin.yml

### Creation checklist

- [X] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

**Comment by Tuditi** [comment]
Completed in https://github.com/iotaledger/firefly/pull/5550
